### DISCUSSION TITLE
language&framework/java/1-jvm-architecture

### DISCUSSION THREAD

**Discussion by giscus** [dev-ujin/dev-ujin.github.io#2] [Comments]
# language&framework/java/1-jvm-architecture

⚓백기선 강사님의 live study에 나온 주제들을 혼자 공부하고 정리하여 작성한 글입니다.⚓

https://dev-ujin.github.io/language&framework/java/1-jvm-architecture

**Comment by devyoujin** [comment]
댓글 테스트: 홧팅! 🐱‍🏍

**Comment by jki503** [comment]
Thanks to you, I learned about JVM easily. Please look forward to my JVM posting!
